catalog.html

```diff
@@ -99,21 +96,12 @@
             </ul>
         </nav>
 
-        <div class="swich-button">
             <button class="switch-light"><img src="./img/switch-btn.png" alt="switch mode"></button>
-        </div>
-
-        <div class="header-button-connect">
-            <ul class="header-connect-list">
-                <li class="discord-btn-item">
-                    <button class="btn-discord-header"><img src="./img/discord-original.png" alt="join us in discord"></button>
-                </li>
-                <li class="connect-btn-item">
-                    <button class="btn-connect-header"><img src="./img/connect-btn.png" alt="connect">connect</button>
-                </li>
-            </ul>
-        </div>
-
+            <div class="header-connect-btn">
+                <button class="btn-discord-header"><img src="./img/discord-original.png" alt="join us in discord"></button>
+                <button class="btn-connect-header"><img src="./img/connect-btn.png" alt="connect">connect</button>
+            </div>
+            
     </header>
     
     <main>
```

Comment:
`            <p><a href="./index.html">HOME</a><span class="colored-text">COLLECTIONS</span></p>
`
need to improve like on catalog-item page

Reply:
`<div class="product-list">`
look like an unnecessary wrapper. It can be removed without affecting the layout. Avoid unnecessary elements in the markup.